include/gepetto/gui/osgwidget.hh

```diff
@@ -58,7 +58,7 @@ namespace gepetto {
         WindowsManager::WindowID windowID () const;
 
 signals:
-        void selected (QString name, QVector3D positionInWorldFrame);
+        void clicked (QString name, QVector3D positionInWorldFrame);
```

Comment:
I didn't fin any reference of this function in the documentation.
If you were thinking about the code, I changed all the code who was using selected.